tests/unit/test_hnswlib.cpp

```diff
@@ -1379,10 +1379,13 @@ TEST_F(HNSWLibTest, testSizeEstimation) {
     size_t size_data_per_element = size_links_level0 + dim * sizeof(float) + sizeof(labeltype);
     test_estimation += n * size_data_per_element;
 
-    size_t estimation = VecSimIndex_EstimateInitialSize(&params);
-    ASSERT_GE(estimation, test_estimation);
+    size_t iestimation = VecSimIndex_EstimateInitialSize(&params);
+    ASSERT_GE(iestimation, test_estimation);
```

Comment:
In second thought, maybe a better test will be comparing the estimation to the actual memory consumption as reported by the allocator (relevant for BF as well).